src/LocationForm.js

```diff
@@ -26,7 +26,7 @@ class LocationForm extends Component {
           </label>
           <button type="submit">Submit</button>
         </form>
-        {console.log(this.props)}
+        location = {console.log(this.props)}
```

Comment:
i think this needs to be tests more.  if its a variable it shouldn't be console logged

Reply:
its ok actually